cdap-app-fabric/src/main/java/co/cask/cdap/app/runtime/ProgramRuntimeService.java

```diff
@@ -87,12 +88,11 @@
   ProgramLiveInfo getLiveInfo(Id.Program programId);
 
   /**
-   * Check if any program that satisfy the given {@link Predicate} is running.
+   * Get all programs that are running.
    * Protected only to support v2 APIs
    *
-   * @param predicate Get call on each running {@link Id.Program}.
    * @param types Types of program to check
-   * returns True if a program is running as defined by the predicate.
+   * returns List of all running programs.
    */
-  boolean checkAnyRunning(Predicate<Id.Program> predicate, ProgramType... types);
+  List<Id.Program> getAllRunningPrograms(ProgramType... types);
```

Comment:
To be consistent with other methods in this interface, it should return `List<RuntimeInfo>` and the method name should be called `listAll`.